doc/Getting-started/index.md

```diff
@@ -65,543 +41,211 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
-![](attach/image2014-7-28 13-52-48.png)
+Than select application for your platform, download sources or binaries and follow instructions provided in application description.
 
-Once the notification is sent, you can see the following output in the application:
+<br>
 
-    [...  INFO  NotificationSystemTestApp]  Received notification {"message": "Hello from Kaa!"} for topic with id X
+<p align="center">
+  <img src="attach/sandbox_first_app_view.png" width="500" height="400">
+</p>
 
-**Congratulations with your first Kaa application!**
+<br>
 
 # Next steps
-To create a real-world IoT solution, you will most likely need to implement more features into your application. Kaa provides you with practically everything you might need. The following overview will help you grasp the scope of Kaa capabilities as well as get familiar with the essential documentation, such as [Programming guide]({{root_url}}Programming-guide) and [Administration UI]({{root_url}}Administration-guide/Tenants-and-applications-management) guide.
-
-**Profiling and grouping**
-During a new endpoint registration, Kaa creates an associated _endpoint profile_ for the endpoint. An endpoint profile is basically some meaningful information about the endpoint which may be useful for specific applications. Profiles may contain things like an OS version, amount of RAM, average battery life, type of network connection, device operation mode – virtually anything. An endpoint profile structure in Kaa is configured using a client-side endpoint profile schema. Based on the defined profile schema, Kaa generates an object model to operate against the client side and handles data marshaling all the way to the database. Whenever the client updates its profile information, the endpoint SDK automatically sends these updates to the server as soon as the connection becomes available.
-
-For programming practice, see [collecting endpoint profiles]({{root_url}}Programming-guide/Key-system-features/Endpoint-profiling/).
-
-The information collected in an endpoint’s profile can be used to group endpoints into independently managed entities called _endpoint groups_. On the back end, Kaa provides a [profile filtering language***]() for defining the criteria for group membership. An endpoint can belong to any number of groups. Grouping endpoints can be used, for example, to send targeted notifications or adjust software behavior by applying group-specific configuration overrides.
 
-For programming practice, see [using endpoint groups]({{root_url}}Programming-guide/Key-system-features/Endpoint-groups-management/).
+After you get acquainted with Kaa Sandbox and its demo applications, you can take one step further: install Kaa into your environment and create your own Kaa applications! 
+We will give you [something to start with]({{root_url}}Programming-guide/Your-first-Kaa-application). 
```

Comment:
Don't like phrase `we will give you something to start with`. Don't know how to make it better, though. `Follow the guide to create your first Kaa application!`?